x/amm/keeper/query_join_pool_estimation_test.go

```diff
@@ -32,6 +32,6 @@ func TestJoinPoolEstimation(t *testing.T) {
 		AmountsIn: sdk.NewCoins(sdk.NewInt64Coin("denom1", 100), sdk.NewInt64Coin("denom2", 200)),
 	})
 	require.NoError(t, err)
-	require.Equal(t, sdk.Coins(resp.AmountsIn).String(), "100denom1,200denom2")
+	require.Equal(t, sdk.Coins(resp.AmountsIn).String(), "100denom1,100denom2")
```

Comment:
can we please add a comment here, why this value should be 100 ?
for example: Pool ratio is 1:1, so join pool will accept 100 even though 200 is passed in input